inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -1939,6 +1940,15 @@ void convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function
             cnnLayer->params[ExecGraphInfoSerialization::ORIGINAL_NAMES] = originalNames;
         }
 
+        if (auto subgraph = ::ngraph::as_type_ptr<ngraph::op::Subgraph>(layer)) {
+            std::string names = "";
+            for (auto& op : subgraph->get_body()->get_ordered_ops()) {
+                names += " \n" + op->get_friendly_name();
```

Comment:
If I remember right we use the comma as a delimiter for the original names.

Reply:
done